introduction.md

```diff
@@ -2,12 +2,8 @@
 
-**You can contribute by opening a Pull Request at the [open source content repo](https://github.com/InsideTheSim/ethmerge.com-content) ❤️**
+The Ethereum "2.0" beacon chain running on Proof-of-stake is already live, [secured by millions of ETH](https://www.coindesk.com/ethereum-2-0-deposit-contract-tops-5-5b-staked-ether) and is [reaching consensus](https://beaconcha.in/) on its own amongst its 100K+ validators since December 2020.
 
-Interdum purus cum dictumst turpis primis varius elit nibh himenaeos fringilla porttitor curabitur laoreet egestas, nascetur penatibus eros lorem senectus sociis eget parturient curae vitae venenatis elementum nullam. Accumsan quam adipiscing fermentum mollis nulla porta morbi orci blandit vel, bibendum elementum pretium sollicitudin aenean nisl maecenas at sed auctor sociosqu, mus fames ligula gravida cubilia facilisis mauris vivamus pellentesque duis, faucibus aliquam quis magna ac augue felis montes nascetur. Dignissim nibh porttitor sed rhoncus himenaeos ligula tristique montes hac semper, massa vitae amet dapibus posuere diam litora pretium. Egestas turpis habitant praesent massa ornare est aenean pulvinar taciti faucibus dictumst, mauris ligula nulla porta fermentum congue neque rutrum inceptos ultricies duis litora, scelerisque dis suscipit phasellus eleifend feugiat accumsan magna vivamus laoreet. Curabitur sed cubilia mattis porta fermentum natoque neque gravida, commodo aliquet viverra tristique at primis cum, eu vitae ullamcorper sem auctor diam sagittis.
-
-Per molestie imperdiet ornare sociis ipsum neque iaculis euismod facilisis, mi integer etiam taciti eget habitasse leo nostra mauris, blandit cum fames turpis conubia cursus placerat magnis. Lacinia elementum cursus integer euismod ligula malesuada litora magnis ac, nunc sapien id ridiculus massa taciti tempor amet himenaeos, eget iaculis ut inceptos primis dictumst quisque turpis. Per lacus hendrerit taciti mattis sodales accumsan, aptent quisque nostra mus potenti, venenatis scelerisque pharetra sociosqu eros. Nisl auctor ad sit rutrum eu inceptos pretium laoreet class, sociis fames potenti mauris velit malesuada molestie turpis augue, quam conubia varius fusce maecenas convallis volutpat massa.
-
-Venenatis inceptos taciti posuere tortor vitae commodo per ligula, imperdiet eu eget ad iaculis maecenas vulputate, et conubia sollicitudin tempus lacinia gravida purus. Dignissim elementum tortor posuere fringilla pharetra mollis ad, vitae elit odio natoque libero diam donec pulvinar, nibh purus tincidunt ante eu rhoncus. Tempus arcu egestas auctor praesent porttitor fames ad, mollis ac duis ante facilisis justo.
\ No newline at end of file
+The Merge refers to the merging of the current PoW chain to the PoS chain and is becoming a very active research & development area for the various teams involved in Ethereum ! 
```

Comment:
This should be unpacked a bit more - This is the gist but the intro will need to have it unpacked a little bit more.